soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -3832,6 +3839,20 @@ void RandomizerSettingsWindow::DrawElement() {
 
                 UIWidgets::PaddedSeparator();
 
+                //Shuffle Master Sword
+                //RANDOTODO: Disable when Start with Master Sword is active
+                // bool disableShuffleMasterSword = CvarGetInteger("gRandomizeStartingMasterSword", 0);
```

Comment:
Looks like this ended up being unused

Reply:
May consider leaving this in for when the Starting Inventory section is fleshed out.